docs/api/audit_logs.rst

```diff
@@ -1675,6 +1693,43 @@ AuditLogAction
         See :attr:`automod_block_message` for more information on how the
         :attr:`~AuditLogEntry.extra` field is set.
 
+    .. attribute:: onboarding_question_create
+
+        An onboarding prompt was created.
+
+        When this is the action, the type of :attr:`~AuditLogEntry.target` is
+        the :class:`Object` with the ID of the onboarding prompt which
+        was created.
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.options`
```

Comment:
From what I can tell, this (and `_update`) is missing `title`, `single_select`, `required`, `in_onboarding`, and `type`

Reply:
You're right… I don't know why I just put that 😅